config/config.go

```diff
@@ -56,8 +56,9 @@ type Config struct {
 }
 
 type CloudConfig struct {
-	AwsCred aws.ConfigAWS `config:"aws.credentials"`
-	GcpCfg  GcpConfig     `config:"gcp"`
+	AwsCred        aws.ConfigAWS `config:"aws.credentials"`
```

Comment:
Do you mind creating a new `AwsCfg` type to wrap all aws-related configurations?